metis-common/metis-common-utils/src/main/java/eu/europeana/metis/utils/CommonStringValues.java

```diff
@@ -24,8 +24,19 @@ public final class CommonStringValues {
 
   public static final String REPLACEABLE_CRLF_CHARACTERS_REGEX = "[\r\n\t]";
 
-  public static final Pattern CRLF_PATTERN = Pattern.compile(CommonStringValues.REPLACEABLE_CRLF_CHARACTERS_REGEX);
+  public static final Pattern CRLF_PATTERN = Pattern.compile(REPLACEABLE_CRLF_CHARACTERS_REGEX);
 
   private CommonStringValues() {
   }
+
+  /**
+   * Sanitized input value from Logging injection attacks(javasecurity:S5145).
+   * <p>Replaces CR and LF characters with a safe value e.g. '_'.</p>
+   *
+   * @param input the input
+   * @return the sanitized input, safe for logging
+   */
+  public static String sanitizeCRLF(String input) {
+    return input == null ? null : CRLF_PATTERN.matcher(input).replaceAll("");
```

Comment:
suggestion: do returning an empty string would be better? `return input == null ? "" : CRLF_PATTERN.matcher(input).replaceAll("");` instead of `null`?

Reply:
I think not, because it's about sanitizing the field(of bad characters) rather than correcting it. Alternatively I would have thrown an exception, but I thought it's the responsibility of the client to have a valid value.
What do you say?

Reply:
mmm. :thinking: I was looking it from the perspective of not returning a `null` to make the function "safe" and getting always `string` the empty string doesn't harm was my thought.

Reply:
leave it  as it is
![afbeelding](https://github.com/user-attachments/assets/71d8df01-4551-4d25-9b5f-24e90fe3b97a)